server.go

```diff
@@ -95,6 +97,22 @@ func runMain() error {
 		return fmt.Errorf("Failed to initialize the smtp server: %v", err)
 	}
 
+	// get the default master extended public key
+	voteKey, err := stakepooldConnMan.DefaultAccountPubKey()
+	if err != nil {
+		return fmt.Errorf("Failed to get master extended public key for the \"defualt\" account: %v", err)
+	}
+	if votingWalletVoteKey, err = hdkeychain.NewKeyFromString(voteKey); err != nil {
+		return fmt.Errorf("Unable to convert the voting wallet master extended public key: %v", err)
```

Comment:
"convert" is a bit confusing.  I suggest "decode" instead as that more accurately reflects what `NewKeyFromString` does.